README.md

```diff
@@ -175,6 +175,10 @@ __Optional.__ `testcafe-browser-provider-electron` suppresses `did-navigate` and
 
 __Optional.__ If `true`, DevTools will be opened just before tests start.
 
+### enableDebug
```

Comment:
I don't think we need an extra option.
You can calculating the module logic depending on Electron version:
* if Electron < 19, use the old behavior (the `--debug-brk=<port>` will be used`).
* if Electron >= 19, use the  `--inspect-brk=${ports[1]}` parameter.

Reply:
I thought about it, but how can I give the electron version?

Reply:
We have a few ideas on how to implement it in another way. We will check them and update this thread as soon we have any news.

Reply:
@miherlosev any news here?

Reply:
if someone needs this update urgently, you can install it from fork, but hope changes will be released soon 
`npm i github:hackolade/testcafe-browser-provider-electron#feat/v0.0.19`